packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -236,6 +242,27 @@ _Props/Attrs_
 -   `markdown` - for retrieving and modifying the markdown value. If the client modifies the markdown value, it will be parsed into a Node using the
     [prosemirror-markdown parser](https://github.com/ProseMirror/prosemirror-markdown/blob/9049cd1ec20540d70352f8a3e8736fb0d1f9ce1b/src/from_markdown.ts#L199).
     The parsed node will then be rendered in the viewer component as rich text.
+-   `external-link` - is a boolean attribute determining whether all absolute links within the viewer component should open in a new tab. By default, all the links will open in the same tab
+    as per the accessibility guidelines on WCAG (links are below). If opening in a new tab, `nimble-icon-up-right-from-square` will be placed right next to all the links
```

Comment:
Do we have any designer guidance or approval on the use of `nimble-icon-up-right-from-square` to mean "this link will open in a new tab"? We should make sure visual designers agree that this is the correct icon to use, and we should also make sure they are aware that we'll ultimately need a way to indicate that a link will go to an external site (#1331 and #1412).

Reply:
Thanks for bringing this up, Molly. 

Initially, Jesse helped us by getting some bandwidth of Rick Ashby for the visual design of rich text components since we have not worked with any visual designers so far. Looks like Rick is on a vacation from Jul 27 - Aug 15.

Are there any other visual designers helping out in #1412 who can quickly help on this, as this will be supporting both issues? 

It will be much helpful if we can align on a fallback strategy for this item, if we couldn't get any quick help from a visual designer based on their priorities.

Reply:
Since `external-link` is removed for the scope of this PR, visual design icon support is also removed.